charts/camunda-platform/charts/optimize/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "tasklist.fullname" . }}-test-connection"
+  labels:
+{{ include "tasklist.labels" . | indent 4 }}
```

Comment:
the should be `optimize.fullname` and `optimize.labels` right?

Reply:
Ah I missed to update the test-connection. Thanks good point.

Reply:
Ah I missed updating the test-connection. Thanks, good point.